src/routes/+page.svelte

```diff
@@ -1,72 +1,56 @@
+  import landingImageWebP from "$lib/assets/commissioner-strain-with-farmer.jpg?webp&quality=75&imagetools";
+  import landingImageWebpMobile from "$lib/assets/commissioner-strain-with-farmer.jpg?webp&quality=75&w=412&imagetools";
+  import landingImageBlurhash, {
+    width as landingImageWidth,
+    height as landingImageHeight,
+    mean as landingImageMean,
+  } from "$lib/assets/commissioner-strain-with-farmer.jpg?blurhash";
+
   import Image, {
     generateSourcesFromFixedSetOfImages,
     type FixedSetOfImages,
   } from "$lib/components/Image";
 
   const images = [
-    { format: "image/jpeg", size: "original", src: sampleImage },
-    { format: "image/jpeg", size: 412, src: sampleImageMobile },
-    { format: "image/webp", size: "original", src: sampleImageWebp },
-    { format: "image/webp", size: 412, src: sampleImageWebpMobile },
+    { format: "image/jpeg", size: "original", src: landingImage },
+    { format: "image/jpeg", size: 412, src: landingImageMobile },
+    { format: "image/webp", size: "original", src: landingImageWebP },
+    { format: "image/webp", size: 412, src: landingImageWebpMobile },
   ] satisfies FixedSetOfImages;
 
   const imageSources = generateSourcesFromFixedSetOfImages(images);
+
+  export let data;
 </script>
 
-<Hero />
-<section class="grid-container usa-section">
-  <div class="grid-row grid-gap">
-    <div class="tablet:grid-col-4">
-      <h2 class="font-heading-xl margin-top-0 tablet:margin-bottom-0">
-        A tagline highlights your approach
-      </h2>
+<Image
+  class="main-image"
+  alt=""
+  src={landingImage}
+  sources={imageSources}
+  blurhash={landingImageBlurhash}
+  width={landingImageWidth}
+  height={landingImageHeight}
+  mean={landingImageMean}
+/>
+<section class="grid-container homepage-greeting maxw-tablet">
+  <ContentfulRichText document={data.title} />
+  <div class="grid-row">
+    <div class="grid-col-4 construction-sign-container">
+      <img
+        class="construction-sign"
+        alt="Under construction sign swinging in the wind"
+        src={underConstructionGif}
+      />
     </div>
-    <div class="tablet:grid-col-8 usa-prose">
-      <p>
-        The tagline should inspire confidence and interest, focusing on the value that your overall
-        approach offers to your audience. Use a heading typeface and keep your tagline to just a few
-        words, and don’t confuse or mystify.
-      </p>
-      <p>
-        Use the right side of the grid to explain the tagline a bit more. What are your goals? How
-        do you do your work? Write in the present tense, and stay brief here. People who are
-        interested can find details on internal pages.
-      </p>
-      <a class="usa-button usa-button--big" href="/">Call to action</a>
+    <div class="grid-col-8 padding-left-4">
+      <ContentfulRichText document={data.body} />
     </div>
   </div>
 </section>
-<GraphicList />
```

Comment:
removing these components from the landing page but keeping them around for the time being